pkg/proxy/winkernel/hnsV2.go

```diff
@@ -30,6 +30,11 @@ import (
 
 type hnsV2 struct{}
 
+var (
```

Comment:
I don't think it's a good idea to hardcode this. This flag should probably be added to hcsshim

Reply:
Yes, this flag value has to be added to HcsShim eventually  but until we have the changes in hcsshim vendored this is the quickest way to get this functionality working in Windows KubeProxy.

Reply:
I agree with @elweb9858 . Please make the appropriate change in hccshim. Changes there can usually be merged fairly quickly and we have some time until the next Kubernetes release.